### DISCUSSION TITLE
Stable release date for biomod2 4.x.x

### DISCUSSION THREAD

**Discussion by Farewe** [biomodhub/biomod2#111] [General]
Hello biomod team,

Do you have an estimated date for stable of biomod 2 4.x.x? 

I give a one-week course on biomod2 every year in January at the Muséum in Paris. Should I consider switching to biomod2 4.x.x for the January 2023 version of the course (will it be stable enough?)? Or should I wait for one more year for a more stable version?

Many thanks
Boris

**Comment by rpatin** [comment]
Hello Boris,
We have started a CRAN submission this week so this should be officially released relatively soon (meanwhile you can use the version on github). We have corrected all major bugs in 4.1-2 that we were aware of. So you can expect version 4.1-2 to be at least as stable as 3.5.1, although you may have to change some habits. Also note that some bugs from 3.5.1 were corrected and some minor functionality added.
Feel free to let us know if you encounter bugs or trouble while adapting your course.
Cheers,
Rémi

**Comment by Farewe** [comment]
Hi Rémi, 

Great, thank you for the update!! And about the news on your ongoing submission to the CRAN. 
I'll open issues if I find significant bugs when preparing my course.

Cheers
Boris

**Comment by rpatin** [comment]
Hi Boris,
Just a quick update to let you know that the new version is now available on CRAN.
Cheers,
Rémi
